yaml/wp_base.yaml

```diff
@@ -0,0 +1,640 @@
+  HEISSGAS_TEMP:                  !include { file: wp_temperature.yaml, vars: { property: "HEISSGAS_TEMP"    }}
+  VERFLUESSIGER_TEMP:             !include { file: wp_temperature.yaml, vars: { property: "VERFLUESSIGER_TEMP"}}
+
+  VORLAUFISTTEMP:                 !include { file: wp_temperature.yaml, vars: { property: "VORLAUFISTTEMP"           , target: "HK1" }}
+  TAUPUNKT_HK1:                   !include { file: wp_temperature.yaml, vars: { property: "TAUPUNKT_HK1"             , target: "HK1" }}
+  RAUMISTTEMP:                    !include { file: wp_temperature.yaml, vars: { property: "RAUMISTTEMP"              , target: "HK1" }}
+  VERSTELLTE_RAUMSOLLTEMP:        !include { file: wp_temperature.yaml, vars: { property: "VERSTELLTE_RAUMSOLLTEMP"  , target: "HK1" }}
+  RAUMSOLLTEMP_I:                 !include { file: wp_temperature.yaml, vars: { property: "RAUMSOLLTEMP_I"           , target: "HK1" }}
+  VORLAUFSOLLTEMP:                !include { file: wp_temperature.yaml, vars: { property: "VORLAUFSOLLTEMP"          , target: "HK1" }}
+  KUEHL_RAUMSOLL_TAG:             !include { file: wp_temperature.yaml, vars: { property: "KUEHL_RAUMSOLL_TAG"       , target: "HK1" }}
+
+  LUEFT_STUFE_TAG:                !include { file: wp_ventilation.yaml, vars: { property: "LUEFT_STUFE_TAG"   }}
+  LUEFT_STUFE_NACHT:              !include { file: wp_ventilation.yaml, vars: { property: "LUEFT_STUFE_NACHT" }}
+  LUEFT_STUFE_PARTY:              !include { file: wp_ventilation.yaml, vars: { property: "LUEFT_STUFE_PARTY" }}
+
+  FEHLERMELDUNG:                  !include { file: wp_generic.yaml, vars: { property: "FEHLERMELDUNG"            , icon: "mdi:alert-circle" }}
+  JAHR:                           !include { file: wp_generic.yaml, vars: { property: "JAHR"                     , icon: "mdi:calendar" }}
+  MONAT:                          !include { file: wp_generic.yaml, vars: { property: "MONAT"                    , icon: "mdi:calendar" }}
+  WOCHENTAG:                      !include { file: wp_generic.yaml, vars: { property: "WOCHENTAG"                , icon: "mdi:calendar-today" }}
+  TAG:                            !include { file: wp_generic.yaml, vars: { property: "TAG"                      , icon: "mdi:calendar-today" }}
+  STUNDE:                         !include { file: wp_generic.yaml, vars: { property: "STUNDE"                   , icon: "mdi:calendar-clock" }}
+  MINUTE:                         !include { file: wp_generic.yaml, vars: { property: "MINUTE"                   , icon: "mdi:calendar-clock" }}
+  ANZEIGE_NIEDERDRUCK:            !include { file: wp_generic.yaml, vars: { property: "ANZEIGE_NIEDERDRUCK"      , accuracy_decimals: "2" }}
+  ANZEIGE_HOCHDRUCK:              !include { file: wp_generic.yaml, vars: { property: "ANZEIGE_HOCHDRUCK"      , accuracy_decimals: "2" }}
+  #PUMPENZYKLEN_MIN_AUSSENT:       !include { file: wp_generic.yaml, vars: { property: "PUMPENZYKLEN_MIN_AUSSENT" }} # THZ404 -> AKTIVE_HEIZSTUFEN
```

Comment:
The whole idea behind the variant handling is that we no longer have to comment out stuff in the yaml to make it work for a specific heat pump ... the only place where we decide which variant to build is in the esp32-poe-technik.yaml
Those specific packages should first go to thz404.yaml ... others can pick it up and test for e.g. thz504. Once proven that it works for all variants we move it back to this file

Reply:
I will also change it here later ;-)